### ISSUE TITLE
Alt Tag

### ISSUE THREAD

**Issue by mikepollard-sbr** [dimitrov-adrian/directus-extension-editorjs-interface#37]
Would love to see alt tag support with the Editor Image. I did see an issue request for Alt tag support (https://github.com/editor-js/image/issues/30), but there hasn't been any move on this.

Would it be possible to extend the ImageTool here to support the Alt Tag?

**Comment by dimitrov-adrian** [comment]
It is not impossible, but as far as I see, it might need to clone the image tool and use local version for it.

But honestly I do not see the use case, since there is a `caption` field, which could hold title/alt and there is a `file.title` and `file.name` that comes from the actual file reference, which also could be used for alts

**Comment by LegendMike-Pollard** [comment]
Makes sense, I can see some cases where the caption and the image being different in some articles. Where the caption text is displayed under the image maybe for copyright purposes. And the alt tag is used for accessibility purposes describing the image.
